code/processes/tickerlogreplay.q

```diff
@@ -211,7 +211,8 @@ replaylog:{[logfile]
          @[`.;`upd;:;.replay.initialupd]];
 	@[`.;`upd;:;.replay.realupd]];
  .replay.tablecounts:.replay.errorcounts:.replay.pathlist:()!();
- .replay.replaydate:"D"$-10#string logfile;
+ // extract date from timestamp if stp, extract date from date if tp
+ $[segmentedmode;.replay.replaydate:"D"$-6_-12#string logfile;.replay.replaydate:"D"$-10#string logfile];
```

Comment:
This works assuming stp log file names will always have timestamp as suffix, but if there is a better way to do this without hardcoded numbers would be interested to know (tabular and none mode now append timestamp so works for all stp log name cases)

Reply:
No I think that is fine. We are in control of the log file format, and we append timestamp to end of all, so it's ok to hardcode a number in here. Is 10 correct though? I thought the format was YYYYMMDDHHMMSS ?

Reply:
The 10 is for the old tp log file format, I think it grabs the date from "yyyy.mm.dd"